### ISSUE TITLE
add support for showing xbox or playstation icon in vcs when spoofing console platform 

### ISSUE THREAD

**Issue by Pickymarker** [AggelosAst/vencord-StatusSpoof#3]
add support for showing xbox or playstation icon in vcs when spoofing console platform

**Comment by AggelosAst** [comment]
Hi, could you show me an example of that? Also, do not forget to check your last issue that you created.

**Comment by AggelosAst** [comment]
if you're talking about people who use the `PlatformIndicators` plugin, they will always see it but not normal users. Discord has yet to add a custom status for that or indication/

**Comment by Pickymarker** [comment]
no I mean when on Xbox or Playstation they have a icon on the right of there name in discord depending on console they are on

**Comment by AggelosAst** [comment]
Are you talking about this?
![image](https://github.com/AggelosAst/vencord-StatusSpoof/assets/87412823/0829e16c-3a21-4eaa-b5e8-7d17d2954711)

**Comment by Pickymarker** [comment]
yeah the Playstation icon or xbox but when in vc everyone would see it next to name if you are on one of them using discord so not just profile also next to name in vcs the icon

**Comment by AggelosAst** [comment]
Im taking a look at it, ill update you on it

**Comment by AggelosAst** [comment]
That is part of the RPCServer service discord uses so i will have to proceed into hooking RPC requests and in the end find the special payload for that.

**Comment by AggelosAst** [comment]
Hi pickymarker, ive been working on the status and i can say i have gotten some progress.
![image](https://github.com/AggelosAst/vencord-StatusSpoof/assets/87412823/00de556a-31cd-4af7-811b-4ff0fbe0514b)
